server/project/test/appointment_handler_test.py

```diff
@@ -175,3 +182,377 @@ def test_start_after_next_x_days(self):
         self.assertEqual(data['message'], f"You may only schedule an "
                                           f"appointment within the next "
                                           f"{NEXT_X_DAYS} days in the future.")
+
+    def test_multiple_appointments(self):
+        """Tests whether a single participant can have multiple appointments."""
+        add_user()
+        db.session.commit()
+        user = User.query.first()
+        user_public_id = user.public_id
+        add_event(user.id, create_availability())
+        db.session.commit()
+        event = Event.query.first()
+        event_url = event.url
+
+        name = 'Fabulous Johnny'
+        email = 'johnny@fabulous.com'
+        start1 = dt.datetime(year=2020,
+                             month=3,
+                             day=2,
+                             hour=9,
+                             tzinfo=dt.timezone.utc)
+        start2 = start1 + dt.timedelta(hours=2)
+        route = f'/users/{user_public_id}/events/{event_url}/appointments'
+        response1 = self.api.post(
+            route,
+            data=create_appointment_json(start=start1.isoformat(),
+                                         name=name,
+                                         email=email),
+            content_type='application/json')
+        response2 = self.api.post(
+            route,
+            data=create_appointment_json(start=start2.isoformat(),
+                                         name=name,
+                                         email=email),
+            content_type='application/json')
+
+        data1 = json.loads(response1.data.decode())
+        data2 = json.loads(response1.data.decode())
```

Comment:
should this be `response2.data.decode()`

Reply:
Fixed